cooltools/saddle.py

```diff
@@ -73,7 +73,7 @@ def digitize_track(binedges, track, regions=None):
         edges. See encoding details in Notes.
     track : tuple of (DataFrame, str)
         bedGraph-like dataframe along with the name of the value column.
-    regions: viewframe, sequence of str or tuples
+    view_df: viewframe, sequence of str or tuples
```

Comment:
No backward compatibility: looks like view_df cannot be a sequence of str or tuples anymore